app/components/atoms/socialMediaButtons/socialMediaButton.component.tsx

```diff
@@ -0,0 +1,77 @@
+import {FC} from 'react';
+import {ButtonProps, StyledButton} from './socialMediaButton.styles';
+
+export const Button: FC<ButtonProps> = ({socialMediaIcon, ...props}) => {
+  let icon;
```

Comment:
would prefer to have a functional switch or even a hash with the configuration better than the entire svgs here and the switch/case here:


something like this (ts pseudo code)
```typescript
const buttonConfig = {
    "instagram": {
        "url": "https://instagram.com/blablabla",
        "svgFile": "instagram.svg",
    }, // ... others
}

const iconProps = buttonConfig[socialMediaIcon]
return (
    <StyledButton {...props}>
        <a href={iconProps['url']}>
            <div className...>
                <img src={iconProps['svgFile']}/>
            </div>
        </a>
    </StyledButton>
)
```